app/models/oral_history_content.rb

```diff
@@ -32,6 +32,15 @@ class OralHistoryContent < ApplicationRecord
   include CombinedAudioUploader::Attachment.new(:combined_audio_mp3, store: :combined_audio_derivatives)
   include CombinedAudioUploader::Attachment.new(:combined_audio_webm, store: :combined_audio_derivatives)
 
+  job_status_options = {
+    queued: 'queued',
+    started: 'started',
+    failed: 'failed',
+    done: 'done'
```

Comment:
"done" is ambiguous, cause failed is a form of done too. Can we please use "succeeded", to go with "failed", as originally proposed?

Reply:
No problem.